### DISCUSSION TITLE
Use BullMQ Worker/Queue in different servers but same Redis

### DISCUSSION THREAD

**Discussion by ragokan** [taskforcesh/bullmq#1224] [Q&A] [answered]
Hello, can I do the approach I mentioned in title?

So, basically, in my server, I will only create **queues** and add jobs to queue. Then I will run a secondary node server that only have **workers**. They will both be connected to the same *Redis* server.

**Comment by manast** [comment, marked as answer]
Yeah. BullMQ is a distributed queue system, you can have as many workers as you want connected to a given Redis host.

**Reply by ragokan** [reply]
Thank you so much for your answer :)

**Reply by Luryy** [reply]
Do I need any additional configuration to implement that?
Or could I simple run the worker file in a server and the queue file in another server?

**Reply by ragokan** [reply]
@Luryy, no need for configuration, they just have to use the same redis instance.
